labs-ios-starter/Model/Apollo/Web Auth/Member.swift

```diff
@@ -6,17 +6,28 @@
 //  Copyright © 2020 Lambda, Inc. All rights reserved.
 //
 
-import Foundation
+import UIKit
 
 
 /// Standard user, leader if topic originator
 struct Member: Equatable, Codable {
 
-    var identifier: UUID?
-    var token: Bearer? //if different from identifier (assign in AuthService)
+    enum CodingKeys: String, CodingKey {
+        case id
+        case email
+        case firstName = "firstname"
+        case lastName = "lastname"
+        case avatarURL = "avatarUrl"
+    }
+
+    var id: String?
```

Comment:
this may need to be `identifier` on our end

Reply:
I think it's fine with lowercase id